src/sidebars/favourites/favourites.less

```diff
@@ -1,11 +1,13 @@
-﻿@favourite-height: 100px;
+﻿@favourite-height: 110px;
 @favourite-padding-left: 0px;
-@favourite-padding-top: 5px;
-@favourite-padding-right: 4px;
-@favourite-padding-bottom: 10px;
+@favourite-padding-top: 10px;
+// Padding right is declared in the variables file.
```

Comment:
Yes I felt similar about it. I wasn't sure what that variable would be called though: I couldn't think of a general yet specific name. 

They could all be moved out, but then is there a need for that?